Makefile

```diff
@@ -35,6 +35,9 @@ REVISION=$(shell git rev-parse HEAD)$(shell if ! git diff --no-ext-diff --quiet
 PACKAGE=github.com/containerd/containerd
 SHIM_CGO_ENABLED ?= 0
 
+# SIGRTMIN is 34 in GNU libc, 35 in musl.
+SIGRTMIN ?= 34
```

Comment:
Can we autodetect libc by `readlink`ing /lib/libc.so ?

Reply:
How about `ldd --version`?

Reply:
I think it is best to have a static value for this.
Ultimately the signal is handled from inside the container and doesn't matter what containerd was linked against.

Reply:
Oh, I wasn't aware that the signal is ultimately handled by a container.